mail-storage/src/main/java/mail/storage/controller/MessageController.java

```diff
@@ -0,0 +1,76 @@
+package mail.storage.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import mail.storage.domain.Message;
+import mail.storage.dto.DateRangeDto;
+import mail.storage.dto.MessageDto;
+import mail.storage.dto.UpdateMessageDto;
+import mail.storage.exception.DraftMessageException;
+import mail.storage.exception.MessageWithNumberAlreadyExists;
+import mail.storage.exception.MessageWithNumberNotFound;
+import mail.storage.service.MailStorageService;
+import org.springframework.cache.annotation.*;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/message")
```

Comment:
I think that i has to be plural /messages

Reply:
Fixed.